security/storage/aws/public_buckets/README.md

```diff
@@ -1,28 +1,42 @@
-## AWS Open Buckets Policy Template
+# AWS Open Buckets
 
-### What it does
+## What it does
 
 This Policy Template will check your account for Amazon S3 buckets with public permission.
 
-### Input Parameters
+## Input Parameters
 
 This policy has the following input parameters required when launching the policy.
 
-- *Email addresses of the recipients you wish to notify* - Example: noreply@example.co
+- *param_email* - Email addresses of the recipients you wish to notify
+- *param_slack_channel* - (Optional) Slack channel name including the \"#\"
+- *param_slack_webhook_cred* - (Optional) Name of RightScale credential that contains the Slack webhook credential
 
-### Policy Actions
+## Policy Actions
 
 The following policy actions are taken on any resources found to be out of compliance.
 
 - Send an email report
 
-#### Slack Channel Notification Support
+## Prerequisites
+
+This policy requires the AWS IAM User Credential. When applying the policy select the appropriate credentials
+from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
+
+The credential must contain the value *AWS* in the Provider field.
+Refer to our documentation for more details on the [Credential Service](https://docs.rightscale.com/credentials/)
+
+## Slack Channel Notification Support
+
 The policy includes optional support to send a notification to a slack channel when an anomaly is detected.
 The policy accepts two optional input parameters to support this capability:
+
 - `Slack Channel Name`: This is the slack channel name, e.g. #policy_alerts
 - `RightScale Credential for Slack Channel Webhook`: This is the name of a RightScale credential in the account that contains the Slack webhook URL.
 
-### Required Permissions
+## Required Permissions
+
+### Required RightScale Roles
```

Comment:
No rightscale roles are needed
remove text "The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. "

Reply:
Updated.